### PR TITLE
#65 - Add contact site

### PR DESCRIPTION
**Pull request by paulinaszkudlarek**
With this PR the page gains contact site.
The site uses Google Forms

image

Mobile:
image
image

### CODE REVIEW

File: frontend/components/Contact/ContactForm.vue [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    <PulseBackgroundAnimation/>
+    <div class="relative mx-auto max-w-7xl p-4 sm:px-6 lg:py-24 lg:px-8">
+      <div class="block space-y-12 md:mx-12 md:flex">
+        <div class="block md:w-1/2">
+          <div class="my-12 space-y-6">
+            <h2 class="text-laravel text-xl font-semibold lg:text-3xl">
+              Napisz do nas
+            </h2>
+            <p class="text-base md:text-lg">
+              Masz pytanie? Chętnie Ci na nie odpowiemy!
```
Comment (EwelinaSkrzypacz) [inline frontend/components/Contact/ContactForm.vue]:
```suggestion
              Chętnie odpowiemy na Twoje pytania.
```

File: frontend/components/Contact/ContactForm.vue [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            </p>
+            <p class="text-base md:text-lg">
+              Jeśli chcesz dołączyć do nas, współtworzyć <strong>Laravel <span class="text-laravel">Poland</span> Meetup</strong> i promować swoją firmę, to daj nam znać - skontaktujemy się z Tobą.
```
Comment (EwelinaSkrzypacz) [inline frontend/components/Contact/ContactForm.vue]:
```suggestion
              Jeśli chcesz dołączyć do nas, współtworzyć <strong>Laravel <span class="text-laravel">Poland</span> Meetup</strong> i promować swoją firmę, to daj nam znać — skontaktujemy się z Tobą. 
```

File: frontend/components/Contact/ContactForm.vue [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+              Masz pytanie? Chętnie Ci na nie odpowiemy!
+            </p>
+            <p class="text-base md:text-lg">
+              Jeśli chcesz dołączyć do nas, współtworzyć <strong>Laravel <span class="text-laravel">Poland</span> Meetup</strong> i promować swoją firmę, to daj nam znać - skontaktujemy się z Tobą. 
+            </p>
+          </div>
+          <form id="contact" onsubmit="" class="relative z-10 text-base" method="POST" target="_blank" :action="url">
+            <div class="space-y-4 py-4 md:px-4">
+              <div>
+                <label :for="emailName" class="block text-left text-zinc-700">Email</label>
```
Comment (EwelinaSkrzypacz) [inline frontend/components/Contact/ContactForm.vue]:
```suggestion
                <label :for="emailName" class="block text-left text-zinc-700">E-mail</label>
```

File: frontend/components/Contact/ContactForm.vue [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+import GoogleIcon from '@/components/Icons/GoogleIcon.vue'
+
+let url = import.meta.env['VITE_CONTACT_FORM_URL'] + '/formResponse'
+let emailName = import.meta.env.VITE_CONTACT_FORM_EMAIL_NAME
+let messageName = import.meta.env.VITE_CONTACT_FORM_MESSAGE_NAME
+
+function handleFormSubmit () {
+  document.getElementById('contact').submit()
+}
+
```
Comment (EwelinaSkrzypacz) [inline frontend/components/Contact/ContactForm.vue]: Delete unnecessary empty lines.

Comment (paulinaszkudlarek) [reply frontend/components/Contact/ContactForm.vue]: Done :)

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by MichalMyskow — APPROVED** [review]

**Pull request merged**